app/controllers/users_controller.rb

```diff
@@ -61,11 +62,17 @@ def update
 
   private
 
+  def managers_only_create_editors!
```

Comment:
I already have an ability that kinda looks like that, but I don't think an ability would be able to prevent a manager from upgrading an editor to a manager. CanCan loads the user the manager is editing and if the user is an editor it allows the changes to go through, but it doesn't check the changes being made to the object. That's my understanding at least.